presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueryFramework.java

```diff
@@ -162,17 +162,13 @@ protected void assertQueryFails(@Language("SQL") String sql, @Language("RegExp")
 
     protected void assertQueryFails(Session session, @Language("SQL") String sql, @Language("RegExp") String expectedMessageRegExp)
     {
-        queryRunner.getExclusiveLock().lock();
```

Comment:
This is already done in #7347. If this one gets merged, that one needs to be closed.